website/src/pages/use-cases/incident-response.tsx

```diff
@@ -0,0 +1,365 @@
+                                >
+                                    Request a demo
+                                </Link>
+                                <Link
+                                    className="btn btn-outline-primary w-md-100"
+                                    to="/get-started"
+                                    title="Try Sourcegraph."
+                                    data-button-style={buttonStyle.outline}
+                                    data-button-location={buttonLocation.hero}
+                                    data-button-type="cta"
+                                >
+                                    Try Sourcegraph now
+                                </Link>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </>
+        }
+    >
+        <ContentSection className="my-lg-5">
+            <div className="row mx-lg-0 mx-4">
+                <div className="d-flex justify-content-center w-100 mt-7 mb-lg-4 mb-0">
+                    <h1 className="text-center font-weight-bold w-100 px-lg-8">
+                        Identify the root cause of an incident and fix it everywhere, fast
+                    </h1>
+                </div>
+                <div className="d-flex flex-column flex-lg-row mt-lg-4 mt-6 mb-6">
+                    <div className="text-center mb-4">
+                        <CrosshairsGpsIcon className="mb-4 text-blurple" size={40} />
+                        <h4 className="font-weight-bold">Assess incidents quickly</h4>
+                        <p>
+                            Pinpoint the code responsible for the incident and find the root cause in your codebase.
+                            Understand the code's functionality to verify the issue.
+                        </p>
+                    </div>
+                    <div className="mx-lg-5 text-center mb-4">
+                        <ClockTimeThreeOutlineIcon className="mb-4 text-blurple" size={40} />
+                        <h4 className="font-weight-bold">Plan your remediation</h4>
+                        <p>
+                            Reduce time to resolution by supplying response teams with actionable details, like links to
+                            all affected code.
+                        </p>
+                    </div>
+                    <div className="text-center mb-4">
+                        <WebIcon className="mb-4 text-blurple" size={40} />
+                        <h4 className="font-weight-bold">Execute your plan globally</h4>
+                        <p>
+                            Fix the root cause and confirm the same issue doesn't reoccur in other areas by locating the
+                            code pattern and automating fixes across your entire codebase.
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </ContentSection>
```

Comment:
Hey Tim, just wanted to highlight a re-usable component I made for these sections in the re-platform ([ThreeUpText.tsx](https://github.com/sourcegraph/about-replatform/blob/main/src/components/ThreeUpText.tsx)). I'm actually bringing it over in my Code Reuse use case PR, you can see it [here](https://github.com/sourcegraph/about/pull/5271/files#diff-6e8818987d5ab4775f0849f1d19812324d344a3cca9588bdad68d8fdd1cc0250). I applied it to the Nutanix page, and 3 other Use Case pages that use it. It could be applied here whenever that branch is merged in! Not dire, just an opportunity to DRY up our layouts.